### PR TITLE
Issue Sign up#2 

### PR DESCRIPTION
**Pull request by reenz**
-Created homepage with sign up form
-Created Makersbnb app
-Stored username in session
-Test for signup
-Created page that welcomes the user after signup

### CODE REVIEW

**Comment by AlexandraGF** [comment]
🐶

File: spec/user_signup_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  scenario "user can sign_up with their personal details" do
+    visit('/')
+    fill_in :username,  with: "reenz"
+    fill_in :email,  with: "reenz@test.com"
+    fill_in :password,  with: "password"
+    click_button 'Sign up'
+    expect(page.status_code).to eq(200)
+  end
+
```
Comment (samuelmaker) [inline spec/user_signup_spec.rb:17]: Should we add another test that tests the username is displayed on the bnb list page?

Comment (bpourian) [reply spec/user_signup_spec.rb:17]: Yes I agree as this is testing to ensure our feature is working

**Comment by bpourian** [comment]
🦑

**Comment by samuelmaker** [comment]
:1st_place_medal: 💯

**Pull request merged**